api/routes/books.js

```diff
@@ -26,6 +44,11 @@ router.get('/:id', async (req, res, next) => {
 
 // You should only be able to create a book if the user is an admin
 router.post('/', async (req, res, next) => {
+  if(!req.user){
+    const error = new Error(`Unauthorized.`)
+    error.status = 401
+    next(error)
+  }
```

Comment:
This small bit of code could then like go to a route-level middleware function like `isLoggedIn`.